cmd/relay/relay.go

```diff
@@ -52,7 +52,7 @@ type Config struct {
 	AutoP2PKey     bool
 	MaxResPerPeer  int
 	MaxConns       int
-	RelayLogLevel  string
+	RelayLogLevel  string // TODO(corver): Rename to LibP2PLogLevel.
```

Comment:
why did you defer this renaming refactor?

Reply:
since it includes a flag rename and we have old relays still running, want to avoid two different flags in production